scipy/fft/_backend.py

```diff
@@ -63,6 +64,15 @@ def set_global_backend(backend):
     -----
     This will overwrite the previously set global backend, which, by default, is
     the SciPy implementation.
+
+    Examples
+    --------
+    We can set the global fft backend:
+
+    >>> from scipy.fft import fft, set_global_backend
+    >>> set_global_backend("scipy")  # Sets global backend
```

Comment:
Should probably say that this is the default.

Reply:
I added a comment.